examples/resources/gateway_config_store.tf

```diff
@@ -0,0 +1,15 @@
+resource "konnect_gateway_config_store" "my_configstore" {
+  name  = "Config Store"
```

Comment:
I think the name for config store does not allow space - getting an error which goes away when space is removed.
`{"code":3,"message":"validation
│ error","details":[{"@type":"type.googleapis.com/kong.admin.model.v1.ErrorDetail","type":"ERROR_TYPE_FIELD","field":"name","messages":["must
│ match pattern '^[\\p{N}\\p{L}.\\-_~]*$'"]}]}`

Reply:
Good catch! I’ll update the example